src/memcached-promisify.js

```diff
@@ -0,0 +1,109 @@
+// forked from https://github.com/flightstats/memcached-promisify
+
+import Memcached from 'memcached';
+
+const MAX_EXPIRATION = 60 * 60 * 24 * 30; // memcached's max exp in seconds (30 days)
+const DEFAULT_HOST = 'localhost:11211';
+
+function cachePromise(client, method, ...args) {
+    return new Promise((resolve, reject) => {
+        client._cache[method](...args, (err, data) => { // eslint-disable-line no-underscore-dangle
+            if (err) {
+                reject(err);
+            } else {
+                resolve(data);
+            }
+        });
+    });
+}
+
+/**
+ * Cache
+ * @constructor
+ * @param {string} [cacheHost] - cache host url
+ * @param {Object} [options] - options passed to memcached
+ *
+ * @example
+ *    new Cache('host', { maxExpiration: 900 });
+ *
+ */
+class Cache {
+    constructor(options = {
```

Comment:
If we're going to pass `{hosts: DEFAULT_HOST}`, what will be the value of `maxExpiration`?